test/src/worker/browser-env/runner/test-runner/index.ts

```diff
@@ -372,41 +380,87 @@ describe("worker/browser-env/runner/test-runner", () => {
         });
 
         describe(`"${BrowserEventNames.runBrowserCommand}" event`, () => {
-            describe("should return error as first callback argument if", () => {
-                it("command does not exists in browser instance", done => {
-                    const browser = mkBrowser_();
-                    (BrowserAgent.prototype.getBrowser as SinonStub).resolves(browser);
-                    const socket = mkSocket_() as BrowserViteSocket;
-                    socketClientStub.returns(socket);
+            describe("call command on element instance", () => {
+                describe("should return error as first callback argument if", () => {
+                    it("command does not exists", done => {
+                        const element = mkElement_();
+                        const browser = mkBrowser_();
+                        browser.publicAPI.$ = sandbox.stub().resolves(element);
 
-                    const expectedErrMsg = '"browser.foo" does not exists in browser instance';
+                        (BrowserAgent.prototype.getBrowser as SinonStub).resolves(browser);
+                        const socket = mkSocket_() as BrowserViteSocket;
+                        socketClientStub.returns(socket);
 
-                    runWithEmitBrowserInit(socket).then(() => {
-                        socket.emit(BrowserEventNames.runBrowserCommand, { name: "foo", args: [] }, response => {
-                            try {
-                                assert.match(response, [
-                                    { stack: sinon.match(expectedErrMsg), message: expectedErrMsg },
-                                ]);
-                                done();
-                            } catch (err) {
-                                done(err);
-                            }
+                        const expectedErrMsg = '"element.foo" does not exists in element instance';
+
+                        runWithEmitBrowserInit(socket).then(() => {
+                            socket.emit(
+                                BrowserEventNames.runBrowserCommand,
+                                { name: "foo", args: [], element },
+                                response => {
+                                    try {
+                                        assert.match(response, [
+                                            { stack: sinon.match(expectedErrMsg), message: expectedErrMsg },
+                                        ]);
+                                        done();
+                                    } catch (err) {
+                                        done(err);
+                                    }
+                                },
+                            );
+                        });
+                    });
```

Comment:
It would be more readable with `await socketEmit(socket, { name: "foo", args: [], element })`, as i mentioned in the previous PR

Reply:
Yeah, I know. I will rewrite them someday.